cps/templates/config_edit.html

```diff
@@ -162,6 +162,36 @@ <h4 class="panel-title">
       </div>
     </div>
     {% endif %}
+    <div class="form-group">
+      <input type="checkbox" id="config_use_github_oauth" name="config_use_github_oauth" data-control="github-oauth-settings" {% if content.config_use_github_oauth %}checked{% endif %}>
+      <label for="config_use_github_oauth">{{_('Use')}} GitHub OAuth</label>
+      <a href="https://github.com/settings/developers" target="_blank" style="margin-left: 5px">{{_('Obtain GitHub OAuth Credentail')}}</a>
```

Comment:
`Credentail` should be `Credential` in both instances.

```suggestion
      <a href="https://github.com/settings/developers" target="_blank" style="margin-left: 5px">{{_('Obtain GitHub OAuth Credential')}}</a>
```